Automattic/jetpack
# Conversation: add block example

_This PR is part of the [follow-up tasks](https://github.com/Automattic/jetpack/issues/18123) after merging `conversation`/`dialogue` blocks._

This PR defines an example to the conversation block. It will be consumed by the block styles, block preview in the inserter, and other places where the block is previewed.

Inserter | Block styles
--------|----------
![image](https://user-images.githubusercontent.com/77539/104018489-9d3f9200-5198-11eb-9361-76124758a5e3.png) | ![image](https://user-images.githubusercontent.com/77539/104018503-a4ff3680-5198-11eb-954f-40824392d243.png)


Fixes #

#### Changes proposed in this Pull Request:

* Conversation: add block example

#### Jetpack product discussion


p1610110339335100-slack-

#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Search the `Conversation` block in the inserter. Confirm you see the preview defined by the example.
* Editing a `Conversation` block, open the block settings sidebar, and confirm the preview of styles are defined by the example

#### Proposed changelog entry for your changes:


n/a
